.github/workflows/licence-checker.yml

```diff
@@ -16,12 +16,11 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v4
     - name: Set up Python
-      uses: actions/setup-python@v2
+      uses: actions/setup-python@v5
       with:
-        # TODO: Check on Python 3.8 too
-        python-version: 3.6
+        python-version: 3.9
```

Comment:
I think we use 3.11 for OT?

Reply:
Yes, but this repo is using 3.9. Should I update everything to 3.11?

Reply:
Will update in a separate PR